configure

```diff
@@ -690,7 +690,11 @@ def configure_library(lib, output):
     if default_libpath:
       default_libpath = '-L' + default_libpath
     (pkg_libs, pkg_cflags, pkg_libpath) = pkg_config(lib)
-    cflags = pkg_cflags.split('-I') if pkg_cflags else default_cflags
+    # Remove empty strings from the list of include_dirs
+    if pkg_cflags:
```

Comment:
@thefourtheye I've never been able to get `pkg-config` give me a similar output. Even if it did, calling strip on `"  "` (two spaces) would return empty:
```python
>>> "  ".strip()
''
```